ui-tests/src/test/resources/features/integrations/integration-import-export.feature

```diff
@@ -117,7 +116,102 @@ Feature: Test functionality of integration import export feature
     Then she waits until integration "Integration_import_export_test" gets into "Active" state
 
 
+#
+#  2. integration-import with drag'n'drop
+#
+  @integration-export-with-drag-and-drop
+  Scenario: Create integration and test export feature of single integration
+
+    # create integration
+    And "Camilla" navigates to the "Home" page
+    And clicks on the "Create Integration" button to create a new integration.
+    Then she is presented with a visual integration editor
+    And she is prompted to select a "Start" connection from a list of available connections
+
+    # select twitter connection
```

Comment:
And review this metastep. It should dramatically reduce the step needed for both variants dragged and non-dragged. 

https://github.com/syndesisio/syndesis-qe/blob/75c655c34227bf8f9b6e687d3596d5e955533dfe/ui-tests/src/test/resources/features/integrations/twitter-to-salesforce.feature#L7-L9